packages/wordpress/components/blockStyles/blocks/pullquote.js

```diff
@@ -0,0 +1,58 @@
+import { createGlobalStyle } from 'styled-components';
+import { siteTheme } from '@irvingjs/styled/utils';
+import { link } from './utils';
+
+/* eslint-disable indent */
+export const PullquoteBlock = createGlobalStyle`
+  .wp-block-pullquote {
+    display: flex;
+    padding: 1rem 1.5rem;
+    text-align: left;
+
+    &::before {
+      color: #AAAAAA;
+      content: '“';
+      font-family: 'Times New Roman', Times, serif;
+      font-size: 4rem;
+      font-weight: 700;
+      margin-right: 1.5rem;
+      margin-top: -5rem;
```

Comment:
Was this top margin intentional? I think this will result in the quote mark being far above the box:
![image](https://user-images.githubusercontent.com/1542278/95354103-0c53d480-0892-11eb-846d-f820d4b33c7f.png)